docs/test-plan.md

```diff
@@ -44,21 +44,21 @@ choose to go through some of the Optional Test Cases.
 
 #### Test case 2: Running a problem query and viewing results
 
-1. Open the [javascript UnsafeJQueryPlugin query](https://github.com/github/codeql/blob/main/javascript/ql/src/Security/CWE-079/UnsafeJQueryPlugin.ql).
+1. Open the [javascript ReDoS query](https://github.com/github/codeql/blob/main/javascript/ql/src/Performance/ReDoS.ql).
```

Comment:
This was probably a copy-paste error from the section above. `UnsafeJQueryPlugin.ql` isn't a normal problem query, and doesn't have results on `babel/babel` 🙈